app/controllers/provider_interface/organisation_permissions_setup_controller.rb

```diff
@@ -0,0 +1,36 @@
+module ProviderInterface
+  class OrganisationPermissionsSetupController < ProviderInterfaceController
+    before_action :require_manage_organisations_permission!
+    before_action :redirect_unless_permissions_to_setup
```

Comment:
We probably also want to redirect to the old flow if the feature flag is off, and vice versa in the old controller